### ISSUE TITLE
Gmail icon spam in search field

### ISSUE THREAD

**Issue by PortStone** [passff/passff#510]
### Versions
- Operating system: Gentoo
- Browser: 91.7.0esr (64-bit)
- PassFF: 1.13
- Host app: 1.2.3

Status line output: `[hh:mm:ss] show -> (0)`

[14:45:40] ls -> (0) no error message

### Observations

- Output of `pass`: 
```
Password Store
└── email
    ├── <redacted>
    └── <redacted>
```
- Your preferences
  - In PassFF: default settings
  - In the host app `passff.py`: default settings
- Messages in the web console: 

---

### Steps to reproduce the issue
1. Go to gmail

### Actual behaviour

Multiple icons from passfs that are not clickable.

![image](https://user-images.githubusercontent.com/27790087/163734103-12a64bb5-83bc-49b0-85cd-4f2cb9348d2a.png)

### Expected behaviour

No icon at all.

**Comment by tuxor1337** [comment]
Duplicate of #449 which is already fixed by #469. So, something must have changed in the meantime...

**Comment by PortStone** [comment]
``` html

<div id="gs_lc50" style="position: relative;">
    <input class="gb_ef" aria-label="In E-Mails suchen" autocomplete="off" placeholder="In E-Mails suchen" value="" name="q" type="text" style="background: transparent url(&quot;moz-extension://36789a99-64ae-4541-ba7b-1e599eb35a66/skin/icon-light.svg&quot;) repeat scroll 0% 0%; cursor: auto; border: medium none; padding: 0px; margin: 0px; height: auto; width: 100%; position: absolute; z-index: 6; left: 0px;" dir="ltr" spellcheck="false" aria-haspopup="true" aria-live="off" aria-owns="gs_sbt50">
    <input disabled="" autocomplete="off" style="border: medium none; padding: 0px; margin: 0px; height: auto; width: 100%; position: absolute; z-index: 1; background-color: transparent; color: silver; left: 0px; visibility: hidden;" id="gs_taif50" class="gb_ef" dir="ltr">
</div>
```

After looking at your fix it seems like the error is simple happening again because I am using Gmail in German. Could we just simple deactivate the extension if someone is on the site mail.google.com to fix it no matter the language is used?

**Comment by tuxor1337** [comment]
Hm, maybe we can ignore fields with `autocomplete="off"`. Hopefully, there are only few login fields out there that use this value in a wasted attempt to prevent spam, or because the devs don't understand the purpose of this attribute.

**Comment by tuxor1337** [comment]
This should be fixed by 4a8a8d4

**Comment by PortStone** [comment]
> This should be fixed by [4a8a8d4](https://github.com/passff/passff/commit/4a8a8d475c6814ffdfe04a6e01be09c0afd067f8)

I would like to test it out. Is this going to be released or should I just clone the repo?

**Comment by PortStone** [comment]
I disabled my installed PassFF extension and installed temporarily PassFF through the git repository. It worked fine and there was no icon at all in the search field like it should be.

**Comment by tuxor1337** [comment]
Thanks a lot! This will be released soon as part of 1.14 :)
